### ISSUE TITLE
UnsupportedOperationException with tooltip on JFileChooser on Linux

### ISSUE THREAD

**Issue by tychobrailleur** [weisJ/darklaf#174]
**Describe the bug**
I have a report of an `UnsupportedOperationException` being thrown on Windows when opening a `JFileChooser`, seemingly due to the tooltip handling:

```
java.lang.UnsupportedOperationException: TRANSLUCENT translucency is not supported.
	at java.desktop/java.awt.Window.setOpacity(Window.java:3647)
	at com.github.weisj.darklaf.ui.tooltip.DarkTooltipUI$FadeInAnimator.paintNow(DarkTooltipUI.java:346)
	at com.github.weisj.darklaf.graphics.Animator.paint(Animator.java:140)
	at com.github.weisj.darklaf.graphics.Animator.onTick(Animator.java:176)
	at com.github.weisj.darklaf.graphics.Animator.access$000(Animator.java:38)
	at com.github.weisj.darklaf.graphics.Animator$1.lambda$run$0(Animator.java:130)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:117)
	at java.desktop/java.awt.WaitDispatchSupport$2.run(WaitDispatchSupport.java:190)
	at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:235)
	at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:233)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:312)
	at java.desktop/java.awt.WaitDispatchSupport.enter(WaitDispatchSupport.java:233)
	at java.desktop/java.awt.Dialog.show(Dialog.java:1070)
	at java.desktop/javax.swing.JFileChooser.showDialog(JFileChooser.java:756)
	at java.desktop/javax.swing.JFileChooser.showSaveDialog(JFileChooser.java:678)
	at core.net.OnlineWorker.askForHRFPath(OnlineWorker.java:1179)
```

Here is the relevant section in `java.awt.Window` :

```java
           if (opacity < 1.0f) {
                GraphicsConfiguration gc = getGraphicsConfiguration();
                GraphicsDevice gd = gc.getDevice();
                if (gc.getDevice().getFullScreenWindow() == this) {
                    throw new IllegalComponentStateException(
                        "Setting opacity for full-screen window is not supported.");
                }
                if (!gd.isWindowTranslucencySupported(
                    GraphicsDevice.WindowTranslucency.TRANSLUCENT))
                {
                    throw new UnsupportedOperationException(
                        "TRANSLUCENT translucency is not supported.");
                }
            }
```

**To Reproduce**

Use a `JFileChooser` with darklaf on a platform whose `GraphicsDevice` does not support:

```java
    public static enum WindowTranslucency {
. . .
        /**
         * Represents support in the underlying system for windows all of
         * the pixels of which have the same alpha value between or including
         * 0.0 and 1.0.
         */
        TRANSLUCENT,
```
(I cannot reproduce on my own system, macOS)

**Screenshots**
N/A

**Additional Information:**
 - OS: Linux
 - OS Version: 
 - Darklaf Version: 2.2.0
 - JDK: 14.0.1

**Additional context**

**Comment by weisJ** [comment]
This problem probably applies to any operating system with a screen device that doesn't support translucency. Also any tooltip should be affected due to the fade-in animation.
I'll try to release a fix as soon as possible.

**Comment by weisJ** [comment]
This is now fixed in `2.1.1`. Thanks for reporting!
